app/components/mandatarissen/update-state.hbs

```diff
@@ -22,23 +22,15 @@
     </div>
   </div>
   <div>
-    <AuLabel>
-      Vanaf
-    </AuLabel>
-    <AuDateInput
+    <DateInput
+      @label="Vanaf"
       @value={{this.date}}
-      @onChange={{this.updateDate}}
-      @error={{not this.isValidDate}}
-      @alignment="default"
+      @from={{this.today}}
+      @to={{@mandataris.einde}}
```

Comment:
i think indeed this is not ok and it should be limited to the start and end of the period

Reply:
(maybe the mandataris doesn't have an end date)

Reply:
Isn't it weird than that we are filling in todays date  as default  when opening the wijzig mandataris?

Reply:
Yes de date input handles a null date as enddate